planner/core/testdata/enforce_mpp_suite_out.json

```diff
@@ -974,5 +974,85 @@
+          "              └─ExchangeSender_29 1.00 mpp[tiflash]  ExchangeType: HashPartition, Hash Cols: [name: test.t.c, collate: binary]",
+          "                └─HashAgg_26 1.00 mpp[tiflash]  group by:test.t.c, funcs:count(test.t.a)->Column#15, funcs:count(1)->Column#16",
+          "                  └─TableFullScan_13 10000.00 mpp[tiflash] table:t keep order:false, stats:pseudo"
+        ],
+        "Warn": null
+      },
+      {
+        "SQL": "EXPLAIN select count(distinct c+a), count(a) from t;",
+        "Plan": [
+          "TableReader_34 1.00 root  data:ExchangeSender_33",
+          "└─ExchangeSender_33 1.00 mpp[tiflash]  ExchangeType: PassThrough",
+          "  └─Projection_27 1.00 mpp[tiflash]  Column#7, Column#8",
+          "    └─HashAgg_28 1.00 mpp[tiflash]  funcs:sum(Column#15)->Column#7, funcs:sum(Column#16)->Column#8",
+          "      └─ExchangeReceiver_32 1.00 mpp[tiflash]  ",
+          "        └─ExchangeSender_31 1.00 mpp[tiflash]  ExchangeType: PassThrough",
+          "          └─HashAgg_28 1.00 mpp[tiflash]  funcs:count(distinct Column#13)->Column#15, funcs:sum(Column#14)->Column#16",
+          "            └─ExchangeReceiver_30 1.00 mpp[tiflash]  ",
+          "              └─ExchangeSender_29 1.00 mpp[tiflash]  ExchangeType: HashPartition, Hash Cols: ",
+          "                └─HashAgg_26 1.00 mpp[tiflash]  group by:Column#18, funcs:count(Column#17)->Column#14",
+          "                  └─Projection_35 10000.00 mpp[tiflash]  test.t.a, plus(test.t.c, test.t.a)->Column#18",
+          "                    └─TableFullScan_13 10000.00 mpp[tiflash] table:t keep order:false, stats:pseudo"
+        ],
+        "Warn": null
+      },
+      {
+        "SQL": "EXPLAIN select sum(b), count(distinct c+a, b, e), count(a+b) from t;",
+        "Plan": [
+          "TableReader_34 1.00 root  data:ExchangeSender_33",
+          "└─ExchangeSender_33 1.00 mpp[tiflash]  ExchangeType: PassThrough",
+          "  └─Projection_27 1.00 mpp[tiflash]  Column#7, Column#8, Column#9",
+          "    └─HashAgg_28 1.00 mpp[tiflash]  funcs:sum(Column#19)->Column#7, funcs:sum(Column#20)->Column#8, funcs:sum(Column#21)->Column#9",
+          "      └─ExchangeReceiver_32 1.00 mpp[tiflash]  ",
+          "        └─ExchangeSender_31 1.00 mpp[tiflash]  ExchangeType: PassThrough",
+          "          └─HashAgg_28 1.00 mpp[tiflash]  funcs:sum(Column#16)->Column#19, funcs:count(distinct Column#17, test.t.b, test.t.e)->Column#20, funcs:sum(Column#18)->Column#21",
+          "            └─ExchangeReceiver_30 1.00 mpp[tiflash]  ",
+          "              └─ExchangeSender_29 1.00 mpp[tiflash]  ExchangeType: HashPartition, Hash Cols: [name: test.t.b, collate: binary], [name: test.t.e, collate: utf8mb4_bin]",
+          "                └─HashAgg_26 1.00 mpp[tiflash]  group by:Column#24, Column#25, Column#26, funcs:sum(Column#22)->Column#16, funcs:count(Column#23)->Column#18",
```

Comment:
what are column#25 and column#26 from the lower layer?

Reply:
It has column # instead of column name before this patch, I think they are fine at the moment.